### PR TITLE
Updates to README pt.2

### PR DESCRIPTION
**Pull request by markbrogan13**
Minor fixes in markdown syntax

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,12 +14,12 @@ the Discord server with custom commands.
 1. Follow steps 1 & 2 from `How to use`
 1. Locate your `Calendar ID` in you Google Calendar settings:
 
-![image](https://user-images.githubusercontent.com/47675634/87125869-0ec97280-c240-11ea-815b-ed13596cef6b.PNG)
+    ![image](https://user-images.githubusercontent.com/47675634/87125869-0ec97280-c240-11ea-815b-ed13596cef6b.PNG)
 
-Select which calendar you want to access through the API.
-Scroll down to `Integrate Calendar` and copy your `Calendar Id`:
+1. Select which calendar you want to access through the API.
```
Comment (hsorathia) [inline README.md:19]: can you remove the "." from the end so the formatting is similar to the `How to use section`?

File: README.md [hunk 2]
```diff
@@ -14,12 +14,12 @@ the Discord server with custom commands.
 1. Follow steps 1 & 2 from `How to use`
```
Comment (hsorathia) [inline README.md:14]: thoughts about just repeating steps 1 and 2 here so they don't have to refer back to another part of the readme?

and then do the same thing w/ the ending steps. i think it actually might read nicer but can u try and see what you think?

**Comment by qsaml** [comment]
clear and concise

File: README.md [hunk 3]
```diff
@@ -11,19 +11,21 @@ the Discord server with custom commands.
 1. Test out the bot in the discord channel!
 
 ## How to set up Calendar Event API
-1. Follow steps 1 & 2 from `How to use`
+1. Obtain API token frome SCE slack
```
Comment (hsorathia) [inline README.md:14]: frome -> from

File: README.md [hunk 4]
```diff
@@ -11,19 +11,21 @@ the Discord server with custom commands.
...
+1. Create a `config.json` file with similar format as `config.example.json`
 1. Locate your `Calendar ID` in you Google Calendar settings:
 
 ![image](https://user-images.githubusercontent.com/47675634/87125869-0ec97280-c240-11ea-815b-ed13596cef6b.PNG)
 
-Select which calendar you want to access through the API.
+Select which calendar you want to access through the API
```
Comment (hsorathia) [inline README.md:20]: Think this might look good as another list item so it'd look like
1. Then select which calendar you want ...

**Pull request merged**